cluster/src/main/scala/io/snappydata/remote/interpreter/SnappyInterpreterExecute.scala

```diff
@@ -250,13 +250,12 @@ object SnappyInterpreterExecute {
         // expand the users list. Can be a mix of normal user and ldap group
         val commaSepVals = users.split(",")
         commaSepVals.foreach(u => {
+          val uUC = u.toUpperCase
```

Comment:
StringUtil.SQLToUpperCase(u) can be used similar to REFRESH_LDAP_GROUP() procedure.